mixed_pond_experimental/daily_code.py

```diff
@@ -0,0 +1,223 @@
+    phi_at = float((1-r)*e*sigma*((T_ak)**4))
+
+    return(phi_at)
+
+#create function for phi_ws water surface radiation
+#this equation need water surface temp T_wk (kelvin), this will be initialized using the t-1 timestep value
+# from the water temp data file (ideally Jan 1st 2018, morning water temp)
+
+def calculate_phi_ws(T_wk, day_argue):
+    
+
+# set T_wk such that it reads the final output water temp from results in a loop
+    
+    phi_ws = float(0.97 * sigma * ((T_wk)**4))
+
+    return(phi_ws)
+
+#create function for phi_e evaporative heat loss
+#here, we use average dailt dew point temp in place of relative humidity values
+
+def calculate_phi_e(T_wk,day_argue):
+    daily_data = read_dataline(day_argue)
+    wind_speed = daily_data['wind_speed_2m']
+
+    W_2 = wind_speed * 3.6
+    air_temp_line = read_air_temp(day_argue)
+    T_ak = air_temp_line['avg_temp'] #kelvin
+    T_ac = T_ak -273.15 #degree celcius
+
+# e_s, saturated vapor pressure needs to be in T_wc deg celcius
+
+    T_wc = T_wk - 273.15
+    e_s = 25.374 * math.exp(17.62 - 5271/T_wc)
+    
+    RH = relative_humidity[(relative_humidity['day']== day_argue)]
+
+    RH = RH['RH2M']
+
+# e_a, water vapor pressure above the pond surface; unit mmHg
+    
+    e_a = RH * 25.374 * math.exp(17.62 - 5271/T_ac)     
+
+    phi_e = float(N* W_2 * (e_s- e_a))
+    return(phi_e)
+
+#create function for phi_c sensible heat transfer
+
+def calculate_phi_c(T_wk, day_argue):
+    daily_data = read_dataline(day_argue)
+    wind_speed = daily_data['wind_speed_2m']
+
+    W = float(wind_speed) #m/s per C&B paper
+    
+    air_temp_line = read_air_temp(day_argue)
+    T_ak = air_temp_line['avg_temp'] #kelvin
+    
+    T_wc = T_wk - 273.15 #convert to deg celcius
+    T_ac = T_ak - 273.15 
+
+    phi_c = float(1.5701 * W * (T_wc-T_ac))
+
+    return(phi_c)
+
+############### Creating simulation loop for daily values #############################
+# Energy Flux equation: phi_net = phi_sn + phi_at - phi_ws - phi_e - phi_c   
+# Heat at t-1 time step: H_t_1 = T_wk * water_heat_capacity * water_density
+# Heat at t: H_t = H_t_1 + phi_net
+# T_w = H_t/ (water_heat_capacity * water_density)
+
+# loop for energy flux equation 
+def main_simulation_loop():
+
+    global T_wk
+    for day_argue in list(range(1, 1095)):
+
+        phi_sn = calculate_phi_sn(day_argue)
+        phi_at = calculate_phi_at(day_argue)
+        phi_ws = calculate_phi_ws(T_wk, day_argue)
+        phi_e = calculate_phi_e(T_wk, day_argue)
+        phi_c = calculate_phi_c(T_wk, day_argue)
+            
+        phi_net = phi_sn + phi_at - phi_ws - phi_e - phi_c 
+
+        print(phi_net)
+
+        H_t_1 = T_wk * water_heat_capacity * water_density
+
+        H_t = H_t_1 + phi_net
+        T_w = H_t/ (water_heat_capacity * water_density)
+
+        #add T_w to a list somehow
+        T_wk_vec.append(T_w)
+
+        T_wk = T_w 
+
+    print(T_wk_vec)
+
+    T_wk_C = np.array(T_wk_vec) - 273.15
```

Comment:
So I think in the calculation of the new temperature we will need Temperature in celcius not in Kelvin for the actual calculation and not just for plotting